app/components/start_page_banner/style.scss

```diff
@@ -1,13 +1,30 @@
 @import "../base.scss";
 @import "../../webpacker/styles/inverted";
 
+.app-start-page {
+  .app-govuk-phase-banner {
+    background-color: $govuk-brand-colour;
+    margin-top: -(govuk-spacing(2));
+  }
+
+  .govuk-phase-banner__text,
+  .govuk-link {
+    color: govuk-colour("white");
+  }
+
+  .govuk-tag {
+    background: govuk-colour("white");
+    color: $govuk-brand-colour;
+  }
```

Comment:
I wonder if it would be better to create new classes for these and apply them directly to the elements on the page.

Reply:
`app-tag--inverted` etc

Reply:
The component is added to the layout file and used on every page so changing the class isn't really an option unless dynamically add different classes based on the page which feels like unnecessary logic that'll need testing etc